web-test-runner.sauce.config.mjs

```diff
@@ -0,0 +1,55 @@
+import config from './web-test-runner.config.mjs';
+import { createSauceLabsLauncher } from '@web/test-runner-saucelabs';
+
+const sauceLabsOptions = {
+	// eslint-disable-next-line no-undef
+	user: process.env.SAUCE_USERNAME,
+	// eslint-disable-next-line no-undef
+	key: process.env.SAUCE_ACCESS_KEY,
+};
+
+const sauceLabsCapabilities = {
+	name: '@brightspace-ui/core unit tests',
+	// eslint-disable-next-line no-undef
+	build: `@brightspace-ui/core ${process.env.GITHUB_REF ?? 'local'} build ${process.env.GITHUB_RUN_NUMBER ?? ''}`
+};
+
+const sauceLabsLauncher = createSauceLabsLauncher(
+	sauceLabsOptions,
+	sauceLabsCapabilities
+);
+
+const extraOptions = {
+	idleTimeout: 500 // default 90
+};
+
+config.concurrentBrowsers = 4; // concurrent browsers
+config.concurrency = 1; // concurrent tests in a single browser
```

Comment:
Huge 🤦  moment: this defaulted to `6` which meant it was running 6 tests at a time in each browser... which was super fast but was also the reason for all the broken tests!

So after two days of unknowingly trying to fix tests so they can run in parallel, just setting this to `1` solves all the problems. It'd be great to run stuff in parallel, and maybe one day we can break our tests out into "isolated" vs. not... but for now I don't think it's realistic as long as we're reusing fixtures across tests for them to be able to run at the same time.